packages/terra-data-grid/src/index.js

```diff
@@ -0,0 +1,5 @@
+import WorklistDataGrid from './WorklistDataGrid';
+
+// TODO: Re-enable linter rule when FlowsheetDataGrid is added.
```

Comment:
Since the package is data-grid, I think it would make sense for the default export to be the DataGrid component. However it is undecided whether or not we should expose component at this time. So we can remove this comment for now.